readme.md

```diff
@@ -192,7 +195,8 @@ As of now, that is Node.js 12.20+, 14.14+, and 16.0+.
 Our projects sometimes work with older versions, but this is not guaranteed.
 
 This project uses [`vscode-languageserver`][vscode-languageserver] 7, which
-implements language server protocol 3.16.
+implements language server protocol 3.16.0.
+It should work anywhere where LSP 3.6.0 or later is implemented.
```

Comment:
This likely now works. I read through the LSP changelog and I don‘t believe we depends on anything new